core/src/HIP/Kokkos_HIP_Space.cpp

```diff
@@ -94,6 +99,15 @@ void DeepCopyAsyncHIP(void* dst, void const* src, size_t n) {
       [&]() { KOKKOS_IMPL_HIP_SAFE_CALL(hipStreamSynchronize(s)); });
 }
 
+bool hip_driver_check_page_migration(int deviceId) {
```

Comment:
Move the definition to an unnamed namespace

Reply:
done in [58675c](https://github.com/kokkos/kokkos/pull/5112/commits/58675c66750254c7c6ff423880042908afbc523e)